chain/network/src/routing/ibf_peer_set.rs

```diff
@@ -1,12 +1,14 @@
 use crate::routing::edge::{Edge, SimpleEdge};
 use crate::routing::ibf_set::IbfSet;
 use borsh::{BorshDeserialize, BorshSerialize};
+use deepsize::DeepSizeOf;
```

Comment:
This should be behind a feature-flag.

P.S. I feel it would be easier to just use a fully-qualified path inside the `derive()` to avoid a need to have a conditional use.

Reply:
> This should be behind a feature-flag.

@frol `near-network` requires `deepsize::DeepSizeOf` to work. That's why it's not under feature flag. However, we may make `near-network` optionally require `deepsize`, but I think this should be addressed by another PR.

> P.S. I feel it would be easier to just use a fully-qualified path inside the `derive()` to avoid a need to have a conditional use.

I agree, this was suggested by @matklad

Reply:
@frol Oh, I realized that I made all packages use `deepsize_feature` optionally. 
Thanks I fixed it.

I also did a PR, where I do it for whole `near-network`: https://github.com/near/nearcore/pull/5436